docs/hoon/library/2o.md

```diff
@@ -2,150 +2,172 @@
 navhome: /docs/
 ---
 
-
 # 2o: normalizing containers
 ### `++jar`
 
+Tile generator. A `jar` is a `map` of `list`.
```

Comment:
I think this should be "mold generator", to match the others. "tile" is a soon-to-be-obsolete internal compile-time representation of a mold.